src/output.cpp

```diff
@@ -953,6 +951,15 @@ std::string string_input_win(WINDOW *w, std::string input, int max_length, int s
             pos += t.length();
             redraw = true;
         }
+        if( callbacks[ch] ) {
```

Comment:
The check for a callback should probably go first. Or is there a reason you allow overriding the escape key, but not the other keys above? This seems at least inconsistent.